docs/content/en/docs/concepts/workloads/_index.md

```diff
@@ -7,21 +7,35 @@ weight: 10
 hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-A Workload contains information about which tasks should be performed during the `preDeployment` as well as
-the `postDeployment`
-phase of a deployment.
-In its state it keeps track of the currently active `Workload Instances`, which are responsible
-for doing those checks for
-a particular instance of a Deployment/StatefulSet/ReplicaSet (e.g. a Deployment of a certain version).
+A Kubernetes
+[Workload](https://kubernetes.io/docs/concepts/workloads/)
```

Comment:
Here maybe I would not reference the kubernetes workloads, but speak about KeptnWorkload, as these contain information about which tasks (pre/post deployment or pre/post evaluation) should be performed

Reply:
done